### PR TITLE
fix: include transitive vulns and fix typos

### PR DESCRIPTION
**Pull request by deepak1725**
- Update Url, 3Scale constants to Prod.
- Few Typos fix
- Moved `synk-token` flag to under `auth` command. 
- Fixes Issue in calculating Transitive Vulnerabilities

### CODE REVIEW

File: analyses/summary/helper_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func data() driver.GetResponseType {
 
 func TestProcessSummary(t *testing.T) {
 	got := ProcessSummary(data(), false)
-	if diff := cmp.Diff(false, got); diff != "" {
+	if diff := cmp.Diff(true, got); diff != "" {
```
Comment (arajkumar) [inline analyses/summary/helper_test.go:33]: I've missed this, but why do you need to use cmp.Diff just for boolean checks? It is okay to use for struct comparision.

File: analyses/summary/helpers.go [hunk 2]
```diff
@@ -45,16 +45,22 @@ func getResultSummary(analysedResult driver.GetResponseType) *StackSummary {
...
+		privateVul := len(dep.PrivateVulnerabilities)
+		if publicVul+privateVul > 0 {
 			processedData.DirectVulnerableDependencies++
 		}
 		processedData.TotalTransitives += len(dep.Transitives)
 		processedData.Severities = getSeverity(dep.PublicVulnerabilities, processedData.Severities)
 		processedData.Severities = getSeverity(dep.PrivateVulnerabilities, processedData.Severities)
-		processedData.PublicVul += public
-		processedData.PrivateVul += private
+		for _, transVul:= range dep.VulnerableDependencies{
```
Comment (arajkumar) [inline analyses/summary/helpers.go:56]: Your pr title must convey this subtle functionality change. Now you are including transitive vulnerabilities right?

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: analyses/summary/helpers.go (hunk 2 above)
Comment (deepak1725) [reply analyses/summary/helpers.go:56]: Please make changes in title whatever you feel appropriate

File: analyses/summary/helper_test.go (hunk 1 above)
Comment (deepak1725) [reply analyses/summary/helper_test.go]: Fixed. Thanks

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**